main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -122,7 +122,7 @@ DataTableCellUI.prototype._render = function() {
       }
       if (spanArr.length != 0) {
         var span = document.createElement('span');
-        span.textContent = spanArr.join(" ");
+        span.innerHTML = spanArr.join(" ").replace(/\n/, '<br />');
```

Comment:
Hey, creating the br tags using the document object may help with this 💯

Reply:
Thanks @elroykanye. You are helping me to resolve this issue. Really appreciated :)